readthedocs/doc_builder/exceptions.py

```diff
@@ -57,6 +57,11 @@ class BuildUserError(BuildBaseException):
         "Ensure your project is configured to use the output path "
         "'$READTHEDOCS_OUTPUT/html' instead."
     )
+    NO_CONFIG_FILE_DEPRECATED = gettext_noop(
+        "Your project doesn't have the required configuration file to build the documentation. "
+        "Add a configuration file to your project to make it build successfully. "
+        "Read more at https://docs.readthedocs.io/en/stable/config-file/v2.html"
```

Comment:
I feel like I've seen this copy somewhere else, it might be worth updating there too. It sounds a bit wrong to me with the verb choice/placement:

```suggestion
        "Your project doesn't have the configuration file required for building documentation. "
        "Add a configuration file to your project to make it build successfully. "
        "Read more at https://docs.readthedocs.io/en/stable/config-file/v2.html"
```